tests/go/GoImportsBearTest.py

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
Why this change? I think you sud file a new issue for this or just include this in a different commit